src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/ProteinsList.tsx

```diff
@@ -42,8 +47,17 @@ const ProteinsList = (props: ProteinsListProps) => {
   const dispatch = useDispatch();
   const { search } = useLocation();
   const proteinIdToFocus = new URLSearchParams(search).get('protein_id');
+  const defaultTranscriptId = useMemo(() => {
+    const defaultTranscripts = defaultSort(props.gene.transcripts);
+    return defaultTranscripts[0].stable_id;
```

Comment:
Just curious to know why we are recalculating this within each view without storing it in redux?

Reply:
From my understanding first of all this should be a temporary fix and when we get the data from thoas we wouldn't need this. Also we thought this will only be needed in the transcript image but i just found out that we needed it in the protein list as well. But still its only 2 place for now. And from what i heard in the planning meeting this morning, twice repeating is still ok :) 
adding @azangru since he might have a better explanation.

Reply:
I agree that the solution is temporary anyway. We might move the code for determining which transcript is default up to the GeneView component, not to repeat it twice. Moving it all the way to redux may be a bit of a hassle; not sure if it's worth it.

Reply:
Thanks @azangru . since it is temporary is it ok to leave it as it is for now and when we need to implement the actual work when the data is available we can look into moving up the parent component. how does that sound @msmanoj ?

Reply:
I think we are ok with repeating. Not the prettiest code, but we know it's changing and are not yet sure how.

Reply:
Cool, let's leave it as it is for now. Thanks 👍